internal/pkg/term/prompt/select_test.go

```diff
@@ -0,0 +1,141 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package prompt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/stretchr/testify/require"
+)
+
+func TestOption_String(t *testing.T) {
+	testCases := map[string]struct {
+		input  Option
+		wanted string
+	}{
+		"should render the value with a tab if there is no hint": {
+			input: Option{
+				Value: "Help me decide!",
+			},
+			wanted: "Help me decide!\t",
+		},
+		"should render a hint in parenthesis separated by a tab": {
+			input: Option{
+				Value: "Load Balanced Web Service",
+				Hint:  "ELB -> ECS on Fargate",
+			},
+			wanted: "Load Balanced Web Service\t(ELB -> ECS on Fargate)",
+		},
+	}
+
+	for name, tc := range testCases {
+		t.Run(name, func(t *testing.T) {
+			require.Equal(t, tc.wanted, tc.input.String())
+		})
+	}
+}
+
+func TestPrompt_SelectOption(t *testing.T) {
+	t.Run("should return ErrEmptyOptions if there are no options", func(t *testing.T) {
```

Comment:
I am wondering if we should also add a test case in which some of the options do not have Hint?